samples/snippets/src/test/java/aiplatform/CreateTrainingPipelineTabularRegressionSampleTest.java

```diff
@@ -87,7 +72,7 @@ public void tearDown()
     // Assert
     String cancelResponse = bout.toString();
     assertThat(cancelResponse).contains("Cancelled the Training Pipeline");
-    TimeUnit.MINUTES.sleep(2);
+    TimeUnit.MINUTES.sleep(3);

```

Comment:
any reason why we need to increase the value here?

Reply:
Yeah, the cancellation response returns right away, but the cancellation may not actually have completed. To avoid receiving an error during tear-down, we wait for cancellation to complete (hopefully) before deletion. (Granted, this creates a race condition that results in shenanigans like this.)

Reply:
Makes sense. For now we can keep this. If we start seeing issues then we need to figure out a better way to address this.